delta_backend/src/ConversionChecker.py

```diff
@@ -223,17 +328,15 @@ def _convertToDose(self, expressionRule, fieldName, fieldValue, summarise, repor
             return fieldValue
         return "" 
 
-    # Change to Lookup
+    # Change to Lookup (loads expected data as is but if empty use lookup extraction to populate value)
     def _convertToLookUp(self, expressionRule, fieldName, fieldValue, summarise, report_unexpected_exception):
+        if isinstance(fieldValue, str) and any(char.isalpha() for char in fieldValue) and not fieldValue.isdigit():
```

Comment:
What's the reason for performing this specific check and not just looking for any non-empty string?